src/ray/core_worker/core_worker.cc

```diff
@@ -2212,22 +2212,53 @@ Status CoreWorker::CancelTask(const ObjectID &object_id,
 }
 
 Status CoreWorker::CancelChildren(const TaskID &task_id, bool force_kill) {
+  std::vector<std::pair<TaskID, Status>> recursive_cancellation_status;
   bool recursive_success = true;
   for (const auto &child_id : task_manager_->GetPendingChildrenTasks(task_id)) {
     auto child_spec = task_manager_->GetTaskSpec(child_id);
-    if (child_spec.has_value()) {
+    if (!child_spec.has_value()) {
+      recursive_success = false;
+      recursive_cancellation_status.push_back(
+          std::make_pair(child_id,
+                         Status::UnknownError(
+                             "Recursive task cancellation failed--check warning logs.")));
```

Comment:
```suggestion
                             "Recursive task cancellation failed. Check warning logs.")));
```

Reply:
lol surprisingly, this is just copied and pasted. But it makes sense to change it